mappings/net/minecraft/network/packet/c2s/play/RenameItemC2SPacket.mapping

```diff
@@ -1,3 +1,5 @@
 CLASS net/minecraft/class_2855 net/minecraft/network/packet/c2s/play/RenameItemC2SPacket
 	FIELD field_13013 itemName Ljava/lang/String;
+	METHOD <init> (Ljava/lang/String;)V
+		ARG 1 name
```

Comment:
Should match the field imo

Reply:
Imo the field should be `name`

Reply:
@DreenDex just resolve this one and I will approve

Reply:
oopsie, I misunderstood

Reply:
Do i need to rename `getItemName` to `getName`? This looks strange to me, and also can broke some things if mojang will decide to add `getName` method to packets 🙄

Reply:
They won't add deobfuscated `getName` methods or even an obfuscated one. Fabric uses intermediary names fyi, feel free to look at fabric wiki for more info. Just rename that method to `getName`